riptide/config/document/command.py

```diff
@@ -96,6 +96,10 @@ def schema_normal(cls):
             If enabled, read the environment variables in the env-files defined in the project (``env_files``).
             Default: True
 
+        [use_host_network]: bool
+            If enabled, the container uses network mode `host`. Overrides network and port settings
```

Comment:
*tries to use network... ?

Reply:
The container will be configured to use network. Maybe he fails but... there is no try ;)